### DISCUSSION TITLE
当前项目是一个比较老系统 使用Springcloud+dubbo （spring-cloud-starter-dubbo）当前生产服务拆分有20个左右。注册中心zk；遇到在doker容器上 新服务发布后 还会掉用到老实例ip上 ；当前这个问题很影响稳定性

### DISCUSSION THREAD

**Discussion by zgq530xyz** [community/community#102823] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

服务发布后 consumer调用到的还是老的实例ip；
![image](https://github.com/community/community/assets/6175596/4c5d21be-d8cc-4cdb-8579-dbbb14b8dec6)
![image](https://github.com/community/community/assets/6175596/7cd786f1-f51c-4d12-9a2f-b85eb843efec)
![image](https://github.com/community/community/assets/6175596/b9dcf056-dba5-498e-a725-7b71ae3200cf)
如图  cachInvokerUrls还是历史的ip 实际zk上看 节点下ip已经注册到节点下

**Comment by queenofcorgis** [comment]
Hi @zgq530xyz , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
